script.js

```diff
@@ -1 +1,186 @@
-//CEOS 19기 프론트엔드 파이팅🔥 ദ്ദി˶ˊᵕˋ˵)
+// 전역
+const todoUl = document.querySelector(".todo-box__todo ul");
+const doneUl = document.querySelector(".todo-box__done ul");
+// Todo, Done 리스트를 저장하는 전역 변수
+let todoArr = [];
+let doneArr = [];
+
+// 오늘 날짜 반환 함수
+const getTodayDate = () => {
+  const today = new Date();
+  const week = ["일", "월", "화", "수", "목", "금", "토"];
+
+  const year = today.getFullYear();
+  const month = ("0" + (today.getMonth() + 1)).slice(-2);
+  const day = ("0" + today.getDate()).slice(-2);
```

Comment:
`String.prototype.padStart` 를 이용해서 구현 할 수도 있을 것 같아요!
```suggestion
  const month = String(today.getMonth() + 1).padStart(2, '0');
  const day = String(today.getDate()).padStart(2, '0');
```